.gitignore

```diff
@@ -132,3 +132,7 @@ ENV/
 
 # pytest
 .pytest_cache/
+
+# docs build
+.build
+_build
```

Comment:
This is already added on line 122, so you can remove these lines. If you'd like to build docs for Red into an ignored folder, you can simply do it in the `docs` sub-directory, and the Makefile is provided for your convenience to build into this folder.

Reply:
so instead of running `sphinx`, I need to run the makefile, right?

Reply:
You don't have to if you don't want to. You can do `sphinx-build . _build/html -bhtml` instead of `make html`.

Reply:
Also note that tox tests builds the docs using `sphinx-build` with the `-W` flag, which treats warnings as errors - hence why the travis build fails if there are any build warnings.

Reply:
The build is in the gitignore file, you're just building the docs in the wrong folder. Please remove these lines.